packages/client/README.md

```diff
@@ -1,232 +1,296 @@
+| ✅      | [Providers](#providers)         | Integrate with a commercial, open source, or in-house feature management tool.                                                     |
+| ✅      | [Targeting](#targeting)         | Contextually-aware flag evaluation using [evaluation context](https://openfeature.dev/docs/reference/concepts/evaluation-context). |
+| ✅      | [Hooks](#hooks)                 | Add functionality to various stages of the flag evaluation life-cycle.                                                             |
+| ✅      | [Logging](#logging)             | Integrate with popular logging packages.                                                                                           |
+| ✅      | [Named clients](#named-clients) | Utilize multiple providers in a single application.                                                                                |
+| ✅      | [Eventing](#eventing)           | React to state changes in the provider or flag management system.                                                                  |
+| ✅      | [Shutdown](#shutdown)           | Gracefully clean up a provider during application shutdown.                                                                        |
+| ✅      | [Extending](#extending)         | Extend OpenFeature with custom providers and hooks.                                                                                |
 
-```typescript
-// global context for static data
-await OpenFeature.setContext({ origin: document.location.host })
+<sub>Implemented: ✅ | In-progress: ⚠️ | Not implemented yet: ❌</sub>
 
-// use contextual data to determine a flag value
-const boolValue = client.getBooleanValue('some-flag', false);
-```
+[Providers](https://openfeature.dev/docs/reference/concepts/provider) are an abstraction between a flag management system and the OpenFeature SDK.
+Look [here](https://openfeature.dev/ecosystem/?instant_search%5BrefinementList%5D%5Btype%5D%5B0%5D=Provider&instant_search%5BrefinementList%5D%5Bcategory%5D%5B0%5D=Client-side&instant_search%5BrefinementList%5D%5Btechnology%5D%5B0%5D=JavaScript) for a complete list of available providers.
+If the provider you're looking for hasn't been created yet, see the [develop a provider](#develop-a-provider) section to learn how to build it yourself.
 
-### Providers:
+Once you've added a provider as a dependency, it can be registered with OpenFeature like this:
 
-To develop a provider, you need to create a new project and include the OpenFeature SDK as a dependency. This can be a
-new repository or included in an existing contrib repository available under the OpenFeature organization. Finally,
-you’ll then need to write the provider itself. In most languages, this can be accomplished by implementing the provider
-interface exported by the OpenFeature SDK.
+```ts
+// TODO the current docs show this returning a promise
```

Comment:
This is a slight deviation from the exiting readme. This looks correct, but I would like to double-check before merging.